### PR TITLE
Added Job Portal Project

### PR DESCRIPTION
**Pull request by abckhush**
Closes: #595 

- **Title** :  ReactJS Project: Job Portal
- **Name:** : Khushi Kalra
- **Idenitfy yourself:** : GSSOC'24

### Describe the add-ons or changes you've made 📃
Added an advanced ReactJS Project: Job Portal which helps users apply to various jobs and find suitable employment.

### How Has This Been Tested? ⚙️
The application was manually tested by navigating through all the pages, performing searches, and verifying the accuracy of search results.

### Screenshots 📷

screenshot webp
Screenshot 2024-07-27 005108

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

### Checklist: ☑️

- [X] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [X] This PR does not contain plagiarized content.
- [X] I have performed a self-review of my own code.
- [X] I have commented my code, particularly wherever it was hard to understand.
- [X] My changes generate no new warnings.

### CODE REVIEW

**Comment by abckhush** [comment]
@Avdhesh-Varshney All issues have been resolved. Kindly merge this.

**Comment by abckhush** [comment]
@Avdhesh-Varshney Kindly merge this.

File: React-JS-Projects/Advanced/Job-Portal/screenshot.webp.png
Comment (Avdhesh-Varshney) [inline React-JS-Projects/Advanced/Job-Portal/screenshot.webp.png:1]: It should be in webp format not in png format.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

**Comment by abckhush** [comment]
@Avdhesh-Varshney Kindly check now :)

File: React-JS-Projects/Advanced/Job-Portal/README.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+<!-- add the screenshot of the project (Mandatory) -->
+![screenshot webp](https://github.com/user-attachments/assets/f8efc58f-c581-4b87-97fb-21480d42d367)
+![image](https://github.com/user-attachments/assets/a5306b3a-07e1-4b77-bee7-19779246e38c)
+
+
+![Line](https://github.com/Avdhesh-Varshney/WebMasterLog/assets/114330097/4b78510f-a941-45f8-a9d5-80ed0705e847)
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+<h4 align='center'>Developed By <b><i>YOUR NAME</i></b> 👦</h4>
```
Comment (Avdhesh-Varshney) [inline React-JS-Projects/Advanced/Job-Portal/README.md:73]: Kindly raise this pr again. Because of large sequence of commit history.

**Pull request closed without merging**